app/uk/gov/hmrc/eoricommoncomponent/frontend/services/subscription/HandleSubscriptionService.scala

```diff
@@ -35,11 +35,11 @@ class HandleSubscriptionService @Inject() (handleSubscriptionConnector: HandleSu
     recipientDetails: RecipientDetails,
     sapNumber: TaxPayerId,
     eori: Option[Eori],
-    emailVerificationTimestamp: Option[ZonedDateTime],
+    emailVerificationTimestamp: Option[LocalDateTime],
     safeId: SafeId
   )(implicit hc: HeaderCarrier): Future[Unit] = {
     val timestampValue =
-      emailVerificationTimestamp.getOrElse(ZonedDateTime.ofInstant(Clock.systemUTC().instant, ZoneOffset.UTC))
+      emailVerificationTimestamp.getOrElse(LocalDateTime.ofInstant(Clock.systemUTC().instant, ZoneOffset.UTC))
```

Comment:
this will return UTC so currently 1 hour behind local BST, is that what is expected?

Reply:
Hmm, this is what I understand.

Previous value from joda time was: `new DateTime(Clock.systemUTC().instant.toEpochMilli, DateTimeZone.UTC)`

So this must be in UTC, but with this change, I lost the Zone in the output. Because we're reusing this as a string.

I'll come back here to ZonedDateTime or just convert to ZonedDateTime before converting to the string. What do you think? Which approach better?

Reply:
hmm, honestly not sure. I'm leaning towards converting to ZonedDateTime before converting to string but would have difficulty arguing why one method is best.
If it helps I prefer convert to ZonedDateTime before converting to string
However, if you or Nandana really prefer the other option then fair enough

Reply:
Done